src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,41 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
```

Comment:
Please destruct movie prop, it will be cleaner
```suggestion
export const MovieCard = ({ movie }) => (
  const {
    imgUrl,
    title,
    ...
  } = movie
```